<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmulatorOptionActions</class>
 <widget class="QWidget" name="EmulatorOptionActions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>128</width>
    <height>32</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="3">
    <widget class="QToolButton" name="toolButtonStore">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Store value</string>
     </property>
     <property name="icon">
      <iconset resource="../qmc2.qrc">
       <normaloff>:/data/img/filesaveas.png</normaloff>:/data/img/filesaveas.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="toolButtonRevert">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Revert to stored value</string>
     </property>
     <property name="icon">
      <iconset resource="../qmc2.qrc">
       <normaloff>:/data/img/undo.png</normaloff>:/data/img/undo.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="toolButtonEnforceDefault">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Forcibly pass the default value (when checked)</string>
     </property>
     <property name="icon">
      <iconset resource="../qmc2.qrc">
       <normaloff>:/data/img/enforce.png</normaloff>:/data/img/enforce.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="toolButtonReset">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Reset to default value</string>
     </property>
     <property name="icon">
      <iconset resource="../qmc2.qrc">
       <normaloff>:/data/img/broom.png</normaloff>:/data/img/broom.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toolButtonReset</tabstop>
  <tabstop>toolButtonRevert</tabstop>
  <tabstop>toolButtonStore</tabstop>
 </tabstops>
 <resources>
  <include location="../qmc2.qrc"/>
 </resources>
 <connections/>
</ui>
